packages/odyssey-storybook/src/components/odyssey-mui/RadioGroup/RadioGroup.stories.tsx

```diff
@@ -133,3 +137,54 @@ export const Error: StoryObj<RadioGroupProps> = {
     errorMessage: "This field is required.",
   },
 };
+
+export const UncontrolledRadioGroup: StoryObj<RadioGroupProps> = {
+  ...Template,
+  args: {
+    defaultValue: "Warp Speed",
+  },
+  play: async ({ canvasElement, step }) => {
+    await step("select controlled radio button", async () => {
+      const canvas = within(canvasElement);
+      const radiogroup = canvas.getByRole("radiogroup") as HTMLInputElement;
+      const radio = canvas.getByLabelText(
+        "Ludicrous Speed"
+      ) as HTMLInputElement;
+      if (radiogroup && radio) {
+        userEvent.click(radio);
+      }
+      expect(radio).toBeChecked();
+      axeRun("select uncontrolled radio button");
+    });
+  },
+};
+
+export const ControlledRadioGroup: StoryObj<RadioGroupProps> = {
+  ...Template,
+  args: {
+    value: "Ludicrous Speed",
+  },
+  render: function C(props) {
+    const [value, setValue] = useState("Ludicrous Speed");
+    const onChange = useCallback((_, value) => setValue(value), []);
+    return (
+      <RadioGroup {...{ ...props, value, onChange }}>
+        <Radio label="Light Speed" value="Light Speed" />
+        <Radio label="Warp Speed" value="Warp Speed" />
+        <Radio label="Ludicrous Speed" value="Ludicrous Speed" />
+      </RadioGroup>
+    );
+  },
+  play: async ({ canvasElement, step }) => {
+    await step("select uncontrolled radio button", async () => {
+      const canvas = within(canvasElement);
+      const radiogroup = canvas.getByRole("radiogroup") as HTMLInputElement;
+      const radio = canvas.getByLabelText("Warp Speed") as HTMLInputElement;
+      if (radiogroup && radio) {
+        userEvent.click(radio);
+      }
+      expect(radio).toBeChecked();
+      axeRun("select controlled radio button");
+    });
```

Comment:
The step name is "select uncontrolled radio button" whereas the axeRun says "select controlled radio button".